puppet/modules/freight/templates/deploy_debs.erb

```diff
@@ -43,6 +47,18 @@ files.each do |file|
 end
 system "/usr/bin/freight-cache -v -c /home/freight/freight.conf apt/#{os}"
 
+foreman_release_pool = "pool/#{os}/#{version}/f/foreman-release"
+foreman_release_dir = "#{ENV['TARGETCACHE']}/#{foreman_release_pool}"
+release_files = Dir.glob("#{foreman_release_dir}/foreman-release_*.deb")
+release_files.sort! {|x,y| File.mtime(x) <=> File.mtime(y)}
+latest_release_file = release_files.last
```

Comment:
```suggestion
latest_release_file = release_files.max_by { |f| File.mtime(f) }
```

Reply:
TIL!

Reply:
https://ruby-doc.org/core-2.7.1/Enumerable.html#method-i-max_by is very nice